src/main/java/org/galaxyproject/dockstore_galaxy_interface/language/GalaxyWorkflowPlugin.java

```diff
@@ -52,6 +51,20 @@ public GalaxyWorkflowPlugin(PluginWrapper wrapper) {
 
   @Extension
   public static class GalaxyWorkflowPluginImpl implements CompleteLanguageInterface {
+    private ObjectMapper mapper = new ObjectMapper();
+
+    /**
+     * This is basically stolen from org.galaxyproject.gxformat2.Lint.
+     * However, that is generated code and cannot be modified so putting it here along with this warning that
+     * the two should probably be in sync.
+     *
+     * @param workflow
+     * @return
+     */
+    public static boolean isGXFormat2Workflow(final Map<String, Object> workflow) {
+      final String wfClass = (String) workflow.get("class");
+      return wfClass != null && wfClass.equals("GalaxyWorkflow");
```

Comment:
If the code stolen from Galaxy's Lint, this is probably OK, but without knowing things overall, so this might overkill, it seems it would be safer to just do `"GalaxyWorkflow".equals(wfClass)`, which handles both null and not risking a ClassCastException on line 65.

Reply:
It's both, but we can improve our version